cmake/GoogleCloudCppDoxygen.cmake

```diff
@@ -89,46 +127,19 @@ function (google_cloud_cpp_doxygen_targets_impl library)
         "${PROJECT_BINARY_DIR}/examples"
         # Any additional includes that the library needs
         ${GOOGLE_CLOUD_CPP_DOXYGEN_EXTRA_INCLUDES})
-    set(DOXYGEN_GENERATE_LATEX NO)
-    set(DOXYGEN_HAVE_DOT NO)
-    set(DOXYGEN_GRAPHICAL_HIERARCHY NO)
-    set(DOXYGEN_DIRECTORY_GRAPH NO)
-    set(DOXYGEN_CLASS_GRAPH NO)
-    set(DOXYGEN_COLLABORATION_GRAPH NO)
-    set(DOXYGEN_INCLUDE_GRAPH NO)
-    set(DOXYGEN_INCLUDED_BY_GRAPH NO)
-    set(DOXYGEN_DOT_TRANSPARENT YES)
-    set(DOXYGEN_MACRO_EXPANSION YES)
-    set(DOXYGEN_EXPAND_ONLY_PREDEF YES)
```

Comment:
These two should be kept next to `DOXYGEN_PREDEFINED`. I assume in this "how to parse the C++ code" section

Reply:
Done.